addons/account_edi/tests/common.py

```diff
@@ -60,10 +60,15 @@ def create_invoice_from_file(self, module_name, subfolder, filename):
         journal_id = self.company_data['default_journal_sale']
         invoice = journal_id.with_context(default_move_type='in_invoice')._create_invoice_from_single_attachment(attachment)
 
+    def get_generated_file(self, invoice):
+        return invoice.edi_document_ids.filtered(lambda d: d.edi_format_id == self.edi_format).attachment_id
```

Comment:
Could be usefull in `account.move`: `_get_edi_document(self, edi_format)` and/or `_get_edi_attachment(self, edi_format)`